projects/sound-open-firmware/build.sh

```diff
@@ -19,12 +19,30 @@
 #
 ################################################################################
 
-cd $SRC/sof/tools/oss-fuzz
-cp corpus/* $OUT/
-rm -rf build_oss_fuzz
-mkdir -p build_oss_fuzz
-cd build_oss_fuzz
+# Environment: Zephyr has its own toolchain selection mechanism, and
+# the oss-fuzz environment generates warnings and confuses things when
+# building some of the (un-fuzzed) build-time tools that aren't quite
+# clang-compatible.
+export ZEPHYR_TOOLCHAIN_VARIANT=llvm
+unset CC
+unset CCC
+unset CXX
+unset CFLAGS
+unset CXXFLAGS
```

Comment:
I may even revert this

Reply:
This isn't being used to build the fuzzed binary, which is grabbing clang off of PATH.   It's there because there's a build-time tool in SOF (that actually isn't used except on real firmware) that doesn't build with clang due to routine warning collisions.  AFAICT @marc-hb fixed this upstream, not sure if that patch merged or not.

Reply:
OK, false alarm, this seems fine.

Reply:
> AFAICT @marc-hb fixed this upstream, not sure if that patch merged or not.

Yes it's merged:
- https://github.com/thesofproject/sof/pull/7647